Source/ACE/Managers/GuidManager.cs

```diff
@@ -45,6 +53,16 @@ public static uint NewPlayerGuid()
         }
 
         /// <summary>
+        /// Returns New Admin Guid
+        /// </summary>
+        /// <returns></returns>
+        public static uint NewAdminGuid()
```

Comment:
I was looking at that earlier - I don't think that code works as none of the updated values are persisted.   I think it was put in more as a stub until we started saving items because every server restart resets those values (i think) I have not spent a lot of time looking at it tbh.